website2/website/versioned_docs/version-0.20.0-incubating/guides-python-topologies.md

```diff
@@ -182,8 +182,7 @@ $ heron submit local \
 Note the `-` in this submission command. If you define a topology by subclassing `TopologyBuilder` you do not need to instruct Heron where your main method is located.
 
 > #### Example topologies buildable as PEXs
-> * See [this repo](https://github.com/streamlio/pants-dev-environment) for an example of a Heron topology written in Python and deployable as a Pants-packaged PEX.
```

Comment:
is there a `website2/versioned_docs/*/topology-development-topology-api-python.md` to also apply changes to?

Reply:
Good point and addressed. Thanks for catching my laziness 😎